mascara/test/index.js

```
@@ -0,0 +1,20 @@
+var fs = require('fs')
+var path = require('path')
+var browserify = require('browserify');
+var tests = fs.readdirSync(path.join(__dirname, 'lib'))
+var bundlePath = path.join(__dirname, 'test-bundle.js')
+var b = browserify();
+
+// Remove old bundle
+try {
+  fs.unlinkSync(bundlePath)
+} catch (e) {}
```

we should report an error